examples/simple_polaris_client.cc

```diff
@@ -30,6 +30,16 @@ DEFINE_string(polaris_api_key, "",
 DEFINE_string(polaris_unique_id, "",
               "The unique ID to assign to this Polaris connection.");
 
+DEFINE_string(
+    polaris_hostname, "",
+    "Specify an alternate hostname to use when connecting to the Polaris "
+    "corrections network. If blank, use the default hostname.");
+
+DEFINE_string(
+    polaris_api_hostname, "",
+    "Specify an alternate hostname to use when connecting to the Polaris "
+    "authentication API server. If blank, use the default hostname.");
```

Comment:
we should add to this comment that changes this is almost never necessary (for the public)